src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -124,11 +124,9 @@ public void perform_valid_query_on_newly_created_aggregated_object() throws Thro
                              .setHost(hostName)
                              .addHeader("content-type", "application/json")
```

Comment:
Make "application/json" a static variable and use it where ever it is used in this file, eclipse can refactor it to a static variable and it will update it in all places.